doc/source/whatsnew/v3.0.0.rst

```diff
@@ -648,6 +648,7 @@ Conversion
 - Bug in :meth:`DataFrame.update` bool dtype being converted to object (:issue:`55509`)
 - Bug in :meth:`Series.astype` might modify read-only array inplace when casting to a string dtype (:issue:`57212`)
 - Bug in :meth:`Series.reindex` not maintaining ``float32`` type when a ``reindex`` introduces a missing value (:issue:`45857`)
+- Bug in :meth:`sanitize_array` was not taking into account pyarrow arrays. (:issue:`56994`)
```

Comment:
Can we change this note to something more user-facing? Maybe:

```
Assigning a PyArrow array now produces a pd.Series with a pd.ArrowDtype
```

Reply:
Of course, thanks !